test.sh

```diff
@@ -7,52 +7,131 @@ GREEN="\e[32m"
 MAGENTA="\e[95m"
 RESET="\e[0m"
 
+# This function will be called when running this script
+function main(){
+    # This function will be called when running this script
+    run_tests_in_development;
+    run_tests_in_production;
+}
 
-# development or production?
-if [[ $(ls -l docker-compose.yml) == *"production.yml"* ]]; then
+function run_tests_in_development() {
+    echo -e "${MAGENTA}"
+    echo -e "=== Development mode ==="
+    echo -e "${RESET}"
+    WEBAPP_CONTAINER="osmaxxwebappdev"
+    CELERY_CONTAINER="osmaxxcelerydev"
+    DB_CONTAINER="osmaxxdatabasedev"
+    COMPOSE_FILE="compose-development.yml"
+    run_tests;
+}
+
+function run_tests_in_production() {
     echo -e "${MAGENTA}"
     echo -e "=== Production mode ==="
     echo -e "${RESET}"
     WEBAPP_CONTAINER="osmaxxwebapp"
     CELERY_CONTAINER="osmaxxcelery"
-else
-    echo -e "${MAGENTA}=== Develpment mode ==="
-    echo -e "${RESET}"
-    WEBAPP_CONTAINER="osmaxxwebappdev"
-    CELERY_CONTAINER="osmaxxcelerydev"
-fi
+    DB_CONTAINER="osmaxxdatabase"
+    COMPOSE_FILE="compose-production.yml"
+    run_tests;
+}
+
+function reset_containers() {
+    echo -e "resetting all containers";
+    dcompose stop -t 0 &> test.log;
+    dcompose rm -f &> test.log;
+    dcompose build &> test.log;
+}
 
+function reset_container() {
+    echo -e "resetting container ${CONTAINER_TO_BE_RESETTED};"
```

Comment:
On [line 128](https://github.com/geometalab/osmaxx/pull/171/files#diff-1b531c5e405e0bc95890d52cf2f2f5b5R128), it's used as an environment variable for only that command, so I guess one could call it a 'named parameter'.